framework/db/oci/Schema.php

```diff
@@ -523,4 +525,25 @@ public function insert($table, $columns)
 
         return $result;
     }
+
+    /**
+     * Sets the isolation level of the current transaction.
+     * @param string $level The transaction isolation level to use for this transaction.
+     * This can be either [[Transaction::READ_COMMITTED]] or [[Transaction::SERIALIZABLE]].
+     * @throws \yii\base\NotSupportedException when unsupported isolation levels are used.
+     * Oracle only supports SERIALIZABLE and READ COMMITTED.
+     * @see http://docs.oracle.com/cd/E11882_01/server.112/e41084/statements_10005.htm#SQLRF01705
+     * @since 2.0.12
+     */
+    public function setTransactionIsolationLevel($level)
+    {
+        switch ($level) {
+            case Transaction::SERIALIZABLE:
+            case Transaction::READ_COMMITTED:
+            parent::setTransactionIsolationLevel($level);
+                break;
+            default:
+                throw new NotSupportedException(get_class($this) . ' only supports transaction isolation levels READ COMMITTED and SERIALIZABLE.');
```

Comment:
I think you're right. In SQLite's class it handles a syntax difference but it's not needed here.